Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -1511,10 +1578,12 @@ void IBATUpdated()
 template <const XCheckTLBFlag flag>
 static TranslateAddressResult TranslateAddress(u32 address)
 {
-  if (TranslateBatAddess(IsOpcodeFlag(flag) ? ibat_table : dbat_table, &address))
-    return TranslateAddressResult{TranslateAddressResult::BAT_TRANSLATED, address};
+  bool wi;
```

Comment:
```suggestion
  bool wi = false;
```

Even if it gets overwritten to ensure it always has an initialized value for its lifespan